src/coreclr/debug/createdump/crashinfo.cpp

```diff
@@ -929,6 +929,24 @@ FormatString(const char* format, ...)
     return result > 0 ? std::string(buffer) : std::string();
 }
 
+//
+// Converts a WCHAR into a std:string containing a UTF-8 encoded string.
+//
+std::string
+ConvertString(const WCHAR* str)
+{
+    if (str == nullptr)
+        return{};
+
+    int len = WideCharToMultiByte(CP_UTF8, 0, str, -1, nullptr, 0, nullptr, nullptr);
+    if (len == 0)
+        return{};
+
+    ArrayHolder<char> buffer = new char[len + 1];
+    WideCharToMultiByte(CP_UTF8, 0, str, -1, buffer, -1, nullptr, nullptr);
```

Comment:
Shouldn't the `-1` for the `cbMultiByte` parameter be `len + 1`?

Reply:
Yes.

Reply:
Bunch of other places too.